commit 3469731469f96c4de0fa4880f5ecc66f4f6f9985 (refs/changes/78/2878/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-10-27T20:21:30+00:00 (1 year, 11 months ago)
    
    OS-6424 dladm show-phys -P speaks gibberish
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/usr/src/cmd/dladm/dladm.c b/usr/src/cmd/dladm/dladm.c
index a339dab76e..c4fe45cf40 100644
--- a/usr/src/cmd/dladm/dladm.c
+++ b/usr/src/cmd/dladm/dladm.c
@@ -3829,6 +3829,7 @@ print_phys_default(show_state_t *state, datalink_id_t linkid,
 	if (status != DLADM_STATUS_OK)
 		goto done;
 
+	bzero(&pattr, sizeof (pattr));
 	(void) snprintf(pattr.link_phys_device,
 	    sizeof (pattr.link_phys_device), "%s", dpa.dp_dev);
 	(void) dladm_media2str(media, pattr.link_phys_media);
